src/site/content/en/react/how-to-use-amp-in-nextjs/index.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: codelab
+title: How to use AMP in Next.js
+authors:
+  - houssein
+subhead: |
+  Try out the two ways that you can add AMP to a Next.js app
+date: 2019-10-17
+draft: true
+glitch: next-amp-start
+related_post: how-amp-can-guarantee-fastness-in-your-nextjs-app
+---
+
+## What will you learn?
+
+This codelab lets you try out the two ways that you can use AMP in a Next.js app.
+Check out [How AMP can guarantee fastness in your Next.js app][guidance] to learn why you
+might want to add AMP support to your Next.js app.
+
+### How to create Hybrid AMP pages {: #hybrid }
+
+{% Aside 'caution' %}
+  The [AMP-only approach](#amponly) is the recommended path for using AMP with Next.js. The
+  Hybrid AMP approach described in this section has a higher maintenance cost than the AMP-only
+  approach because it requires you to maintain two versions of each page. You should only use the
+  Hybrid approach if you're certain that the AMP-only approach won't work.
+{% endAside %}
```

Comment:
This new aside makes it pretty clear that the Hybrid approach is not recommended